src/test/java/uk/gov/companieshouse/overseasentitiesapi/utils/PopulateUpdateSubmissionTest.java

```diff
@@ -89,11 +89,11 @@ private void checkStatementDetailsArePopulated(UpdateSubmission updateSubmission
 
     private void checkFilingForDataDetailsArePopulated(UpdateSubmission updateSubmission) {
         assertNotNull(updateSubmission.getFilingForDate());
-        assertEquals(String.valueOf(LOCAL_DATE_TODAY.getDayOfMonth()),
+        assertEquals(toStringDate(LOCAL_DATE_TODAY.getDayOfMonth()),
```

Comment:
I would suggest creating a date which can be single digits (ex:1st Feb 2022) so that we can test that day and month is converted 01 and 02 rest

Reply:
Agree with Jitin, let's use a fixed value

Reply:
I think unfortunately the fields/elements are populated by the PopulateUpdateSubmission class as well, so would alter consistency, but in this case, with the month we are going from 6 to 06, is this okay?

Reply:
nvm, will change, shouldn't matter much